prow/config/jobs/api.yaml

```diff
@@ -1,11 +1,33 @@
 org: istio
 repo: api
 support_release_branching: true
-image: gcr.io/istio-testing/build-tools:master-2019-12-05T22-22-12
+image: gcr.io/istio-testing/build-tools:master-2019-12-09T23-37-05
 
 jobs:
   - name: build
     command: [make, presubmit]
 
   - name: gencheck
     command: [make, gen-check]
+
+  - name: gen_client-go
+    type: postsubmit
+    command:
+    - /home/prow/go/src/istio/test-infra/tools/automator/automator.sh
```

Comment:
I have no opinion here: is it better to have the full path, or `../test-infra`?

Reply:
I went with *absolute* to be explicit but *relative* path should be equally as good. I will update to relative since it is cleaner (and it looks like there is a typo either ways since `istio/test-infra` should be `istio.io/test-infra`).

Reply:
Updated. As far as best practices ... I do not think there is one. Whatever is the most intuitive is best and on second thought that is probably relative.

Reply:
Like I said before I don't mind either way, but I could see an argument that relative is preferred because /home/prow/go could hypothetically change. In practice, it won't

Reply:
That is a good point. Yes, while we do try to remain backwards-compatible there is no guarantee the root go path will not change.